v2/internal/config/vars.go

```diff
@@ -111,7 +111,8 @@ var _ fmt.Stringer = Values{}
 // Returns the configuration as a string
 func (v Values) String() string {
 	return fmt.Sprintf(
-		"SubscriptionID:%s/TenantID:%s/ClientID:%s/PodNamespace:%s/OperatorMode:%s/TargetNamespaces:%s/SyncPeriod:%s/ResourceManagerEndpoint:%s/ResourceManagerAudience:%s/AzureAuthorityHost:%s/UseWorkloadIdentityAuth:%t",
+		"SubscriptionID:%s/TenantID:%s/ClientID:%s/PodNamespace:%s/OperatorMode:%s/TargetNamespaces:%s/SyncPeriod:%s"+
+			"/ResourceManagerEndpoint:%s/ResourceManagerAudience:%s/AzureAuthorityHost:%s/UseWorkloadIdentityAuth:%t",
```

Comment:
Would this be better, for readability, if it created a buffer and wrote each name/value pair into that buffer individually?

Reply:
It's probably slower, but yeah it's a bit clearer. I've changed.